app/components/pages/Dashboard/Dashboard.js

```diff
@@ -1,90 +1,78 @@
-import ManuscriptStatus from '../../ui/atoms/ManuscriptStatus'
+import SmallParagraph from '../../ui/atoms/SmallParagraph'
+import ExternalLink from '../../ui/atoms/ExternalLink'
 import StickyFooter from '../../ui/atoms/StickyFooter'
+import Tabs from '../../ui/molecules/Tabs'
+import DashboardList from '../../ui/molecules/DashboardList'
+import { ALL_MANUSCRIPTS } from './oporations'
+import ManuscriptsQuery from './ManuscriptsQuery'
 
-/* Temporary dashboard view pending receipt of designs */
-
-const BoxHiddenFromMobile = styled(Box)`
+const RightAlignedButton = styled(Box)`
+  margin: ${th('space.3')} 0 ${th('space.1')};
+  text-align: right;
   display: none;
   ${media.mobileUp`display: block`};
 `
 
+const DashboardPanel = styled(Tabs.Panel)`
+  min-height: 300px;
+`
+
 const MobileOnlyStickyFooter = styled(StickyFooter)`
   display: block;
   ${media.mobileUp`display: none;`};
 `
 
-const Header = styled.th`
-  text-align: left;
-  padding: 3px 6px;
+const EmptyListMessage = styled.div`
+  margin-top: ${th('space.7')};
+  text-align: center;
+  color: ${th('colorTextSecondary')};
 `
 
-const Cell = styled.td`
-  border-top: 1px solid grey;
-  padding: 3px 6px;
+const CenterdSmallParagraph = styled(SmallParagraph)`
+  color: ${th('colorTextSecondary')};
+  text-align: center;
 `
 
-const Dashboard = ({ manuscripts, deleteManuscript, createManuscript }) => (
-  <React.Fragment>
-    <FormH2>Dashboard Dummy Page</FormH2>
-
-    <BoxHiddenFromMobile mb={4}>
-      <Button
-        data-test-id="desktop-new-submission"
-        onClick={createManuscript}
-        primary
-      >
-        New Submission
-      </Button>
-    </BoxHiddenFromMobile>
-
-    <FormH3>All manuscripts</FormH3>
-
-    {!manuscripts.length && <p>No manuscripts to display</p>}
-
-    {!!manuscripts.length && (
-      <table data-test-id="manuscripts">
-        <thead>
-          <tr>
-            <Header>Title</Header>
-            <Header>Stage</Header>
-            <Header />
-          </tr>
-        </thead>
-        <tbody>
-          {manuscripts.map(manuscript => (
-            <tr key={manuscript.id}>
-              <Cell data-test-id="title">
-                <Paragraph data-test-id="title">
-                  <Link
-                    to={`${manuscript.formState ||
-                      `/submit/${manuscript.id}/author`}`}
-                  >
-                    {manuscript.meta.title || '(Untitled)'}
-                  </Link>
-                </Paragraph>
-              </Cell>
-              <Cell data-test-id="stage">
-                <ManuscriptStatus statusCode={manuscript.clientStatus} />
-              </Cell>
-              <Cell>
-                <Button onClick={() => deleteManuscript(manuscript.id)} small>
-                  Delete
-                </Button>
-              </Cell>
-            </tr>
-          ))}
-        </tbody>
-      </table>
-    )}
+const Dashboard = ({ createManuscript }) => (
```

Comment:
Could we add a markdown file for this component so that the dashboard appears in styleguidist? :pray:

Reply:
I wasn't able to add this to a markdown as it needs the Apollo wrapper for the `ManuscriptsQuery` components. I did some refactoring of the code though and decided this could all be shifted into the main index.js as it is the core 'page' file now.  Please review the structure changes I'm about to push up and let me know if it makes sense or if it was better to have this `Dashboard` component